tools/ab_test.py

```diff
@@ -136,7 +136,9 @@ def load_data_series(report_path: Path, revision: str = None, *, reemit: bool =
                 else:
                     # If there are many data points for a metric, they will be split across
                     # multiple EMF log messages. We need to reassemble :(
-                    assert processed_emf[dimension_set].keys() == result.keys()
+                    assert (
```

Comment:
commit message `Improve message if 1:1 mapping of metrics<->dimensions is broken`: can add `test` tag.

Reply:
The tag would push the commit title beyond 70 characters

Reply:
discussed offline, it's at author's discretion